packages/teleport/src/services/joinToken/types.ts

```diff
@@ -20,3 +20,10 @@ export type JoinToken = {
 };
 
 export type Roles = 'App' | 'Node' | 'Db';
+
+export type Method = 'token' | 'ec2' | 'iam';
+
+export type Rule = {
+  awsAccount?: string;
```

Comment:
🤷 
```suggestion
  awsAccountID?: string;
```

Reply:
Thanks, I changed it to `awsAccountId`, with lower case 'd' since it seems this is how we do in the codebase.